include/cmetrics/cmt_compat.h

```diff
@@ -23,6 +23,7 @@
 #include <time.h>
 #ifdef _WIN32
 #include <windows.h>
+#define strtok_r strtok_s
```

Comment:
we don't use strtok_x() in the project, its very risky:

https://benpfaff.org/writings/clc/strtok.html

Reply:
However, as far as I could implement, strtok_r is reentrant and it could be safe:
freebsd: https://github.com/lattera/freebsd/blob/401a161083850a9a4ce916f37520c084cff1543b/lib/libc/string/strtok.c#L52
apple open source: https://opensource.apple.com/source/Libc/Libc-997.1.1/string/FreeBSD/strtok.c.auto.html

Should we implement strspn or another C runtime function to implement for splitting the tokens?

Reply:
Or, should we use strcpsn/strspn to search segments which is not including `,`. That's also fine.

Reply:
Done by: https://github.com/fluent/cmetrics/pull/210/commits/7dae62a7c32ef554bdb966f703be1557e54bb170